lib/kovid/cli.rb

```diff
@@ -6,6 +6,10 @@
 module Kovid
   class CLI < Thor
     FULL_FLAG = %w[-f --full].freeze
+    
+    def self.exit_on_failure?
+      true
+    end
```

Comment:
You defined this already. No?

Reply:
I don't exactly know what I did, I re-submitted the the PR